languages/server.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable consistent-return */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-console */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable linebreak-style */
+
+import express from 'express';
+import consolidate from 'consolidate';
+import cookieParser from 'cookie-parser';
+
+export class Server {
+  constructor({ i18nProvider, localeService }) {
+    this.express = express();
```

Comment:
so language has it's own server?

Reply:
I've changed it,we're now having the same server